### DISCUSSION TITLE
Add scrollbar on tags list

### DISCUSSION THREAD

**Discussion by djordans** [rustdesk/rustdesk#5760] [Feature Request] [answered]
Hello,

Please add scrollbar on tags list
![Capture d’écran 2023-09-20 110058](https://github.com/rustdesk/rustdesk/assets/5314498/cdb920cf-f852-4ac5-a89c-0151402b8e43)

Thanks

**Comment by rustdesk** [comment]
@21pages

**Reply by 21pages** [reply, marked as answer]
#5762
